jsonnet/jsonnetfile.lock.json

```diff
@@ -142,8 +142,8 @@
           "subdir": "jsonnet/prometheus-operator"
         }
       },
-      "version": "2388bfa557c9836d5ff01e620a129d33852670ff",
-      "sum": "eHJp7oFWvBEsSmwoRML356DLK80n7rRt8XKRZ+YawvQ="
+      "version": "3f047bc9c74105ba774aff726798a66b81409f7f",
+      "sum": "TcXPVhn0BIFCf5mWa6hyIljVl13ItwPjyeiwro56kjE="
```

Comment:
this is pulling new CRD versions that aren't supported by the version of prometheus-operator currently shipped

Reply:
Reverted this change.